app/src/main/java/org/wikipedia/page/ToCHandler.java

```diff
@@ -164,6 +165,8 @@ private void scrollToSection(@Nullable Section section) {
         if (section == null) {
             return;
         }
+        bridge.execute(JavaScriptActionHandler.prepareToScrollTo(section.getAnchor(), "{ highlight: false }"));
+
         if (section.getId() == ABOUT_SECTION_ID) {
             bridge.execute(JavaScriptActionHandler.scrollToFooter(webView.getContext()));
         } else {
```

Comment:
Do we still need this other scrolling logic after this? It sounds like we're making it scroll twice.

Reply:
To clarify: the "scroll_to_anchor" handler will automatically scroll to the desired section anyway. So why do we need to do `scrollToSection()` here a second time?